package/icu/Config.in

```diff
@@ -22,6 +22,16 @@ config BR2_PACKAGE_ICU_USE_ICUDATA
 
 endif
 
+if BR2_PACKAGE_ICU
+
+config BR2_PACKAGE_ICU_FILTERS_DATA_PATH
+	string "Path to filters data file"
+	help
+	  This option allows to define the path to a filters data
+	  file.
+	  Leave empty to not use this functionality.
+endif
```

Comment:
By default, it will increase the size of zImage (with ramfs) significantly.
As there is no default filter defined, it will generate full icu data  e.g. it will increase icudata size by 22MB and zImage size by 10MB. 
Please cross-check if this size increase is OK.